docs/dev/src/en/books/essentials/getting-started/set-up-an-ide/index.md

```diff
@@ -42,14 +42,19 @@ Many instructions on this site assume that you have added `~/reach` to `PATH`.
 
 1. Install [Visual Studio Code](https://code.visualstudio.com/).
 
-1. Add the following file association to vscode preferences:
+    1. Add the following file association to [vscode preferences](https://www.gyanblog.com/vscode/visual-studio-code-associate-file-extensions-language/):
```

Comment:
Okay, I'm about 90% sure on this, but our Visual Studio extension may do this automatically, already.

See [line 165 of `vsce/client/src/extension.ts`](https://github.com/reach-sh/reach-lang/blob/8d9353671570d9d3d30dc62ddb2a233984869d02/vsce/client/src/extension.ts#L165).

So, we might be able to remove step 1 altogether, right, @jeapostrophe?